### ISSUE TITLE
content: '\\'; causes parse error

### ISSUE THREAD

**Issue by jdufresne** [jakubpawlowicz/clean-css#878]
CSS file:

```
p {
    content: '\\';
}
```

This file is considered valid by the [W3C CSS validator](https://jigsaw.w3.org/css-validator/validator).

When running cleancss, it outputs an unexpected warning:

```
$ ./node_modules/.bin/cleancss --version
4.0.0
$ ./node_modules/.bin/cleancss -o base.min.css base.css 
WARNING: Missing '}' at base.css:4:0.
```

I believe this did not happen before 4.0.

**Comment by jakubpawlowicz** [comment]
Yup, something is wrong here.

**Comment by jakubpawlowicz** [comment]
It's fixed in 4.0.4!

**Comment by jdufresne** [comment]
Thanks for the quick fix!
